tests/SocketTest.cpp

```diff
@@ -2,14 +2,69 @@
 #define CATCH_CONFIG_MAIN
 #include "catch.hpp"
 
-#include "pony/Address.hpp"
-#include "pony/Socket.hpp"
+#include <pony/Socket.hpp>
+
+#include <ostream>
 
 using namespace pony;
 
-TEST_CASE("Socket dummy test", "[]")
+TEST_CASE("Invalid initialization", "[Socket]")
 {
+    SocketsInit();
+
     Socket socket;
 
     REQUIRE(false == socket.IsOpen());
+
+    SocketsKill();
+}
+
+const uint16_t port = 33333;
+const uint32_t localhost = (0x1 | 127 << 24);
```

Comment:
yeah, these magic numbers gotta go. Please write a `IPv4Address` generator

Reply:
it will be done in Utility class for uses and tests cases - ok?